InnoGotchi.MVC/InnoGotchi.MVC/Controllers/FarmController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using AutoMapper;
+using InnoGotchi.MVC.Contracts.Services;
+using InnoGotchi.MVC.Models.Farm;
+using InnoGotchi.MVC.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+
+namespace InnoGotchi.MVC.Controllers
+{
+    public class FarmController : Controller
+    {
+        private readonly IFarmService _farmService;
+        private readonly IPetService _petService;
+        private readonly IMapper _mapper;
+
+        public FarmController(IFarmService farmService, IPetService petService, IMapper mapper)
+        {
+            _farmService = farmService;
+            _petService = petService;
+            _mapper = mapper;
+        }
+
+        private (string, string) GetRequiredParameters()
+        {
+            var jwt = Request.Cookies["jwt"];
+            var userId = User.Claims.First(c => c.Type == "Id").Value;
+            return (jwt, userId);
+        }
+
+        public async Task<IActionResult> FarmOverview()
```

Comment:
It's better to add HttpPut/HttpPost, etc. attributes to the endpoints

Reply:
added